applications/FSIapplication/python_scripts/fsi_analysis.py

```diff
@@ -0,0 +1,338 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.MeshMovingApplication as MeshMovingApplication
+import KratosMultiphysics.FluidDynamicsApplication as FluidDynamicsApplication
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+try:
+    import KratosMultiphysics.ExternalSolversApplication
+except ImportError:
+    pass
+
+from analysis_stage import AnalysisStage
+
+class FSIAnalysis(AnalysisStage):
+    '''Main script for FSI simulations using the FSI family of python solvers.'''
+
+    def __init__(self, model, project_parameters):
```

Comment:
why don't you call the baseclass constructor?

Reply:
Because the current structure of the FSI application ProjectParameters.json is not compatible with it. The point is that we need to change the GiD interface writing script for such FSI app, Since this is not a minor task, we decided to keep it as is done in this PR as a temporary solution,